triton-vm/src/state.rs

```diff
@@ -895,8 +893,8 @@ impl<'pgm> VMState<'pgm> {
 impl<'pgm> Display for VMState<'pgm> {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         match self.current_instruction() {
-            Ok(instruction) => {
-                let row = self.to_processor_row(instruction);
+            Ok(_) => {
+                let row = self.to_processor_row();
                 write!(f, "{}", ProcessorMatrixRow { row })
```

Comment:
I'm tempted to get rid of `ProcessorMatrixRow` since we only use it for pretty-printing the VM state, which we could just do with a function.